ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -148,6 +148,32 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private-cco-manual-security-token-service
+- as: aws-ipi-disconnected-network-mtu-localzone-p2-f14
```

Comment:
The profile is `aws-ipi-disconnected-private-network-mtu-localzone`. I think we are supposed to reflect the profile name in test name

Reply:
Avoid name length restriction, like test name `aws-ipi-disconnected-cco-manual-sts-p3-f28-destructive` -> `aws-ipi-disconnected-private-cco-manual-security-token-service` in https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml#L138-L150 

And in Prow CI, all _disconnected_ clusters are _private_, so there is no ambiguity.